_pages/dev/ssr/server-side-rendering-in-spartacus.md

```diff
@@ -21,26 +21,28 @@ The following steps describe how to manually add SSR support so that your Sparta
 1. Add the following dependencies to `package.json`:
 
     ```json
-    "@angular/platform-server": "~8.2.14",
-    "@nguniversal/express-engine": "^8.1.1",
-    "@nguniversal/module-map-ngfactory-loader": "^8.2.6",
+    "@angular/platform-server": "~10.1.0",
+    "@nguniversal/express-engine": "^10.1.0",
     "express": "^4.15.2"
     ```
 
 1. Add the following developer dependencies to `package.json`:
 
     ```json
-    "ts-loader": "^5.3.2",
-    "webpack-cli": "^3.3.2"
+    "ts-loader": "^6.0.4",
+    "webpack-cli": "^3.3.12",
```

Comment:
Not sure if webpack-cli is still required. Didn't saw it in the latest schematics installation. Any thoughts?

Reply:
indeed it's not required anymore